core/src/components/input/input.ionic.outline.scss

```diff
@@ -121,14 +121,8 @@
 
 // This makes the label sit above the input.
 :host(.label-floating.input-fill-outline.input-label-placement-stacked) .label-text-wrapper {
-  @include transform(translateY(0), scale(#{$form-control-label-stacked-scale}));
   @include margin(0);
-
-  /**
-   * Label text should not extend
-   * beyond the bounds of the input.
-   */
-  max-width: calc((100% - var(--padding-start) - var(--padding-end)) / #{$form-control-label-stacked-scale});
```

Comment:
This is no longer needed without the `transform` as a rule already exists to set the `max-width` here:

https://github.com/ionic-team/ionic-framework/blob/bf8dfdb5b1596e2880c6fdbc20559cbb1231649a/core/src/components/input/input.ionic.outline.scss#L76